apps/api/src/campaign/campaign.controller.ts

```diff
@@ -25,7 +25,10 @@ export class CampaignController {
   }
 
   @Post('create-campaign')
-  @Public()
+  @Roles({
+    roles: [RealmViewSupporters.role, ViewSupporters.role],
+    mode: RoleMatchingMode.ANY,
+  })
```

Comment:
These actually are the admin roles and for this endpoint we want all authenticated users to be able to suggest a campaign.

To achieve that we may need only to delete the `@Public()` so the default auth guard mechanism kick in and require authenticated user.